### ISSUE TITLE
Console error when we visit a design challenge detail page directly

### ISSUE THREAD

**Issue by LieutenantRoger** [topcoder-platform/community-app#4171]
**Describe the bug**
Console error happened when we visit a design challenge detail page directly in the browser

**To Reproduce**
1. Visit https://www.topcoder.com/challenges/30120715
2. Open up the console and see the errors

**Expected behavior**
The console should not report error

**Screenshots**
https://monosnap.com/file/XnyxVVwSmnmYu1s4mfeJ9iPrFp1e83

**Additional context**
The work should be done against `hot-fix-console-error`

**Comment by LieutenantRoger** [comment]
Hi @urwithat 

Can you see this error ?  It happens when we visit this API with not enough params. 

`https://api.topcoder-dev.com/v4/challenges/?filter=status%3DACTIVE%26technologies%3D&limit=50&offset=0`


See the error report:

![image](https://user-images.githubusercontent.com/2875203/78451038-7c9c5600-76b5-11ea-894a-39ed8977cf1d.png)

 and the query detail:

![image](https://user-images.githubusercontent.com/2875203/78451049-9a69bb00-76b5-11ea-825a-691f5cd92af1.png)

**Comment by LieutenantRoger** [comment]
Putting this as `On hold` as this looks like a v4 API issue, we are migrating the challenges API to v5, so once the migration is done, we will try to fix it against v5 API.

**Comment by Oanh-and-only-Oanh** [comment]
@LieutenantRoger 1) is this still an issue? 2) if it's still an issue, what's the impact on the end user?

**Comment by LieutenantRoger** [comment]
@Oanh-and-only-Oanh  From the recording, it looked like the V4 API doesn't accept the scenario without technologies params.  But when we try the challenge link provided, there is no error with V5.

The impact is the challenge detail can't be seen if I remember correctly.
